qutebrowser/completion/models/urlmodel.py

```diff
@@ -39,7 +39,11 @@ class UrlCompletionModel(base.BaseCompletionModel):
     def __init__(self, parent=None):
         super().__init__(parent)
 
+        self.columns_to_highlight.append(0)
+        self.columns_to_highlight.append(1)
```

Comment:
Is there a reason you don't just do `self.columns_to_highlight = [0, 1]` here (and in other places)?